src/extension.ts

```diff
@@ -75,7 +99,14 @@ export function activate(context: ExtensionContext) {
 	context.subscriptions.push(disposable);
 }
 
-function registerHiePointCommand(langClient:LanguageClient,name:string,command:string,context:ExtensionContext) {
+function isStackHieInstalled(): Promise<boolean> {
+	return new Promise((resolve, reject) => {
+		const cmd: string = 'stack exec -- which hie'
```

Comment:
I should probably not run this through stack until the scripts use stack as well.